v3/src/components/case-table/collection-table.tsx

```diff
@@ -356,7 +358,7 @@ export const CollectionTable = observer(function CollectionTable(props: IProps)
         <DataGrid ref={gridRef} className="rdg-light" data-testid="collection-table-grid" renderers={renderers}
           columns={columns} rows={rows} headerRowHeight={+styles.headerRowHeight} rowKeyGetter={rowKey}
           rowHeight={+styles.bodyRowHeight} selectedRows={selectedRows} onSelectedRowsChange={setSelectedRows}
-          columnWidths={columnWidths.current} onColumnResize={handleColumnResize} onCellClick={handleCellClick}
+          columnWidths={dataGridColumnWidths} onColumnResize={handleColumnResize} onCellClick={handleCellClick}
```

Comment:
Please double check this. `columnWidths.current` was passed to `DataGrid` previously, but you removed it. So now it's not being used for anything. Leaving it in the code slows the application down and makes the code more confusing for future developers.

Reply:
I hope to get to reviewing this fully later today, but for now -- I added the API between RDG and CODAP around column widths so that serialization of column widths would work. If you remove that API I expect serialization to break.

Reply:
We certainly don't need two maps of column widths. I'll be adding a commit to address this.